### DISCUSSION TITLE
Strange address in Slingbox_server command line

### DISCUSSION THREAD

**Discussion by barbill28** [GerryDazoo/Slinger#660] [General]
Gerry, Frst I'd like to thank you for all the work you've put into this, much appreciated.

I've been using your SlingServer since before they shutdown the online servers. I'm running on a Windows 11 Pro. There has been a strange address (25.77.154.182) in the Slingbox_server command line that I haven't been able to figure out where it's coming from and believe it was there before the latest 4.0 & 4.01 updates. It's a British address and just seems a bit strange. It's not in the config.ini and was hoping you could bring some light on this. Also, looks like I'm getting a double Port (32450) under the remote configuration in the command line (Streamer Running:  10 config.ini SLINGBOX 32450 32450 8388608) and it makes me add that additional port at the end of the remote http address ex: (http://192.168.200.100:32450/Remote/32450) or it won't connect. I believe that may only happen when using the HTML remote but it's been a long time since I used a .txt file I can't be sure. Maybe it's something in the HTML file but I'm not familiar with it so keep my hands off. Also, I don't have any problems internally but when I travel away, the sound will start to breakup and have to hit pause/play to pause and then again to restart to correct. Usually lasts 15-20min and then have to repeat. The remote location has 200MB down and 6MBup. I use VLC for the stream both locally and externally. The slingbox is connected directly to my router and I have GB up/down through FIOS.  Below I'll include the Config.ini as well as my server command line:

Config.ini:

[SLINGBOX]
sbtype="350/500/M1/M2"
;sbtype="Solo/Pro/ProHD"
;sbtype="240"
password=RemovedForSafety

;uncomment the next lines ipaddress and port with your local network info
;if you don't want the server to automatically find your 
;slingbox on the local network. Needed if server and slingbox are
;not on the same LAN segment. Also removes requirement for the netifaces module  
;ipaddress=192.168.200.120
;port=5201

;Replace with your finderid, If and only if you need remote access and don't  
;have a static ip and you plan on using my service to access you server remotely. 
;Please read the release notes. Most people don't need this.
finderid=RemovedForSafety

;If you've got more than one slingbox set name. THe server will use this when generating 
;logs to make it easier to see what's going on. 
;name=something

;Following are the default values
;Default resolution when server starts. You can change this and reconnect without having to restart the server
;valid range = 0..16 
Resolution=16

;Valid Values 1, 6, 10, 15, 20, 30, 60   depending on resolution
FrameRate=60

;Min 50 Max 8000
VideoBandwidth=8000

;Min 0 (auto) max 63
VideoSmoothness=63

;Send Iframe every n seconds Max 30
;IframeRate=5

;Audio Bit Rate. Valid Options 16, 20, 32, 40, 48, 64, 96 Default 32
;AudioBitRate=32

;Video Source  0, 1, 2, 3 depending on your hardware corresponds to one of
;Composite, Component, S-Video, HDMI or Tuner.
;Pro:        0=(Analog)Tuner   1=Composite   2=S-Video     3=Component (Via HDMI to Component Cable)
;ProHD:      0=Tuner           1=Composite   2=S-Video     3=Component
;Solo:       0=Composite       1=S-Video     2=Component
;500:        0=Composite       1=Component   2=HDMI
;350/M1/M2:  0=Composite       1=Component
; If you don't set this, the code will use the last configured input
; I recommend setting this value. If not set correctly, you'll often only see a black screen.
; Note: Setting invalid options may cause unexpected hardware behavior. For example, setting
;  2 or 3 for M2 hardware causes it to reboot.
VideoSource = 1

[SERVER]
;local port number for the server to listen on for connections
port=32450
;maxstreams=10
;enableremote=yes

[REMOTE]
;see release notes for changing this if your remote doesn't work, But as a start make it 
;the same as your configured VideoSource 0-3. See above.
code=1
;path to an external remote control definitions file, if the defaults
;aren't working for you or you want to change the format of the web page.
;use the supplied remote.txt as a starting point.
include=M:\Slingbox\FIOS_remote.html

Server Command Line:

Version : 4.01 Running on Windows-10-10.0.22631-SP0 pid= 19872 M:\Slingbox\slingbox_server.exe
Using config file config.ini
Connection Manager Running on port 32450 with 10 max streams using URL slingbox.
Building page for Slingbox
Reading Custom Remote definition from M:\Slingbox\FIOS_remote.html
Starting Streamer Thread for Slingbox
Streamer Running:  10 config.ini SLINGBOX 32450 32450 8388608
starting up on  port 32450 8388608
32450 Checking for slingbox at ('', 5201) 0
 * Serving Flask app 'slingbox_server'
 * Debug mode: off
32450 Error connecting to  ('', 5201)
Broadcast Responder Running
Server IP address is 192.168.200.100
Server IP address is 25.77.154.182
32450 No valid slingbox ip info found in config.ini
32450 Finding Slingboxes on local network. My IP Info =  192.168.200.100
32450 Found at 192.168.200.120 5201 " Slingbox 500 " FinderID RemovedForSafety ProductID 500
32450 Finding Slingboxes on local network. My IP Info =  25.77.154.182
32450 Using slingbox at  ('192.168.200.120', 5201)
Streamer:  32450 Waiting for first stream, flushing any IR requests that arrive while not connected to slingbox

**Comment by GerryDazoo** [comment]
25.77.154.182 is the IP address of an interface on your PC. Open a command prompt and type ipconfig to list all your interfaces.

**Reply by barbill28** [reply]
SOB! Never thought to check that. I use Hamachi and it's the interface for that. A bit embarrassing since I worked in the IT field for many years. I retired 3yrs ago and it's amazing how fast I've forgotten stuff. Once again, Thanks!
